packages/gitea/build.sh

```diff
@@ -22,6 +21,15 @@ termux_step_make() {
 	cp -a "$TERMUX_PKG_SRCDIR" "$GOPATH"/src/code.gitea.io/gitea
 	cd "$GOPATH"/src/code.gitea.io/gitea
 
+	go mod init || :
+	go mod tidy
+
+	for f in "$GOPATH"/pkg/mod/github.com/lib/pq@*/user_posix.go; do
+		chmod 0755 "$(dirname "$f")"
+		chmod 0644 "$f"
+		sed -i '/^\/\/ +build /s/ linux / linux,!android /g' "$f"
```

Comment:
Could you add a comment about what the sed command does to make it easier to understand? 
My sed-knowledge is a bit weak, but I guess it replace `linux` with `linux,!android` if the line starts with `// +build`?